packages/client/src/lib.js

```diff
@@ -392,7 +392,7 @@ class NFTStorage {
     if (blob.size === 0) {
       throw new Error('Content size is 0, make sure to provide some content')
     }
-    return packCar([{ path: 'blob', content: blob.stream() }], {
+    return packCar([toImportCandidate('blob', blob)], {
```

Comment:
Not an issue with this PR, but does having the path as 'blob' and using `pack` from ipfs-car with it's default `wrapWithDirectory:true` mean this has been creating lots of DAGs of a directory with a single entry of `blob`? If not, why not? is 'blob' used as a magic marker, or is `wrapWithDirectory` being set to `false` somewhere?